UnraisinYourself/ViewController.swift

```diff
@@ -1,21 +1,51 @@
-//
-//  ViewController.swift
-//  UnraisinYourself
-//
-//  Created by Hannie Kim on 7/20/23.
-//
-
 import UIKit
 
 class ViewController: UIViewController {
```

Comment:
Do you mind adding `final` to ViewController since I don't think we'll be subclassing it :)